laravel/app/Http/Controllers/ScheduleDataController.php

```diff
@@ -469,306 +380,156 @@ public function updateMilestones($id){
             ->orderBy('last_name')
             ->get();
 
-        $datas = compact('data1', 'data2', 'data3');
+        $datas = $data;
 
         return view('schedules.resident.milestone', compact('id', 'milestones', 'datas', 'anesthesiologists'));
     }
 
-	public function notifyResidentOverwrittenPreferences($toName, $toEmail, $residentName, $date, $overwrittenChoices)
+    public function notifyResidentOverwrittenPreferences($toName, $toEmail, $residentName, $date, $overwrittenChoices)
```

Comment:
You can probably remove the `$toName` parameter and just set it inside the method to be `''`  instead of having the method call on line 719 have send that in. Since it is the only place that method is used I think it might be a reasonable change despite it making the method less flexible

Reply:
I think `$residentName` is the better choice, since I'm pretty sure the value is (or at least, could be) used when rendering the email.